woocommerce/woocommerce-android
# [HACK] Improvements to performance of the main screen

Closes: #10333


### Description


* Before, we were loading plan information without showing anything when the “more” screen was opened for the first time during the app lifecycle. That looked like laggy. Now, I show the screen right away and load the plan information later.
* Added animation of showing/hiding toolbar

### Testing instructions

* Open more screen -> notice faster opening of that
* Tab between tabs -> notice animation of hiding/showing of the toolbar 

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/fddf7515-aa60-405c-8e88-437457e55555
